app/overrides/layouts/decidim/_wrapper.rb

```diff
@@ -2,8 +2,8 @@
 
 Deface::Override.new(virtual_path: "layouts/decidim/_wrapper",
                      name: "add_google_tag_manager_to_body",
-                     insert_before: "div.off-canvas-wrapper",
-                     original: "5609b164ff19d67a2787c4ead8908da8e7a493d1") do
+                     insert_before: ".layout-container",
+                     original: "e9d156be0184f0a4576b5c7790a8c0e2d2e2ac73") do
```

Comment:
this is also being manually inserted via override here: app/views/layouts/decidim/admin/_header.html.erb

Reply:
what do you mean?

Reply:
See
- app/views/layouts/decidim/_head_extra.html.erb
- app/views/layouts/decidim/admin/_application.html.erb
- app/views/layouts/decidim/admin/_header.html.erb

don't we have repeated code?